server/controllers/entryController.js

```diff
@@ -139,8 +139,40 @@ class EntryController {
         }
     }
 
-    static viewEntry = (req, res) => {
-        
+    static viewEntry = async (req, res) => {
+        try {
+            let {
+                entryId
+            } = req.params
+            
+            const parsedEntryId = parseInt(entryId);
+            const retrieveUserId = returnUserId(req.header('authorization'));
+
+            const values = [ parsedEntryId, retrieveUserId ]
+
+            const getEntriesQuery = `SELECT * FROM mydiaryentries WHERE entryid=$1 AND userid=$2;`;
+            const getUserEntries = await pool.query(getEntriesQuery, values);
+            // return console.log(getUserEntries.rows[0].entryid);
+
+            if (!getUserEntries.rows[0]) {
+                return res.status(400).send({
```

Comment:
@Ntare22 let replace `400` to `404` status code as we are dealing with resource not found instead of a bad request.